src/main/java/de/picturesafe/search/spring/configuration/DefaultQueryConfiguration.java

```diff
@@ -52,6 +54,11 @@ public QueryConfiguration queryConfiguration() {
         return new QueryConfiguration();
     }
 
+    @Bean
+    public QuerystringPreprocessor querystringPreprocessor() {
+        return new NoOpQuerystringPreprocessor();
```

Comment:
I would configure the StandardQueryStringPreprocessor and provide a configuration option, something like 'querystring_preprocessor.enabled=false'

Reply:
done